### ISSUE TITLE
Crashes while authentification on Linux

### ISSUE THREAD

**Issue by freaktechnik** [nightingale-media-player/soundcloud#2]
## STR:
- Install extension
- Click on icon in lower left corner
- Log in with your soundcloud account
- watch it crash while loading the OAuth thingie
- ???
- PROFIT
## Expected result:

Loads OAuth dialog
## Reality:

Crashes.

**Comment by ilikenwf** [comment]
Which OS, Linux?

Works fine for me, occasional crashes but I think it may be from the search constantly loading more results.

**Comment by freaktechnik** [comment]
No, it's while it's loading the auth dialog, the popover. Ubuntu/elementary OS (@GeekShadow and me had the issue).

**Comment by ilikenwf** [comment]
Firefox dep issue? Got a stack trace or debug build output?
